featuretools/computational_backends/feature_set.py

```diff
@@ -135,6 +136,8 @@ def _add_feature_to_trie(self, trie, feature, approximate_feature_trie,
         sub_ignored_trie = approximate_feature_trie.get_node(feature.relationship_path)
 
         for dep_feat in feature.get_dependencies():
+            if isinstance(dep_feat, MultiOutputFeature):
```

Comment:
that would break other usages of `get_dependencies`, particularly in the `from_dictionary` usages, for that function should return a specific slice or multi output feature with the associated column number, not just the base feature of the multi output without the column